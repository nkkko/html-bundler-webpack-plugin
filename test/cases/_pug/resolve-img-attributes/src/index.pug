html
  head
    title Test
  body
    h1 Hello World!
    img(src='@images/apple.png')
    img&attributes({
      class: 'image',
      src: '@images/fig.png',
      srcset: '@images/pear.png 300w, @images/plum.png 600w',
      alt: 'picture',
    })
