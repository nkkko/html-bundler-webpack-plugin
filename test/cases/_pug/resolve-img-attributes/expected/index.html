<!DOCTYPE html><html><head><title>Test</title></head><body><h1>Hello World!</h1><img src="apple.02a7c382.png"><img class="image" src="fig.c6809878.png" srcset="pear.6b9b072a.png 300w, plum.d39e7174.png 600w" alt="picture"></body></html>